src/app/home/form/form.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { FormComponent } from './form.component';
```

Comment:
Try to delete these files while pushing or not creating them,command for this would be `ng g c --skip-tests=true {{component_name}}`. Here `{{ }}` is used for string-interpolation,it could be replaced by appropriate component-name

Reply:
ok